wgsl/index.bs

```diff
@@ -442,6 +442,19 @@ literal_or_ident
     and [=fragment=] inputs.
     See [[#interpolation]].
 
+  <tr><td><dfn noexport dfn-for="attribute">`invariant`
+    <td>*None*
+    <td>Must only be applied to an entry point return type, or member of a
+    [=structure=] type.
+    Must not be applied to user-defined IO (see [[#user-data-attributes).
+
+    When applied to the `position` [=built-in output variable=] of a vertex
+    shader, the computation of the result is invariant across programs.
+    That is, if the data and control flow match for two `position` outputs in
+    different entry points, then the result values are guaranteed to be the
+    same.
+    Has not effect when applied to any other built-in variable.
```

Comment:
why do we want to allow this if it has no effect?

Reply:
This is the stance we've taken for many other attributes. Also, thinking of sharing structs, it needs to be allowed on position input at least for fragments. We could say only apply to position built-in variables I suppose, but it is less symmetric with other attributes.

Reply:
right. Our stance on other attributes is dictated by the need to use the relevant structs/types in other contexts. Here, there is no use of invariance outside of the position builtin, for now at least. So we should only allow it there. If we want to relax this later, we can.